RealmLoginKit Android/app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
```

Comment:
I don't see how to comment on those .png files.  Shouldn't we use our logo or something, for the launcher icon?